src/utils/loadConfig.ts

```diff
@@ -8,23 +8,37 @@ import RawExportRule from '../types/rawConfig/RawExportRule';
 import ConfigSet from '../types/ConfigSet';
 import ExportRule from '../types/config/ExportRule';
 import validateRawConfig from '../validation/validateRawConfig';
+import NormalizedPath from '../types/NormalizedPath';
 
-export default function loadConfig(file: string, configSet: ConfigSet) {
+export function loadConfigFromString(
```

Comment:
Looks like this function doesn't need to be exported.

Reply:
It _will_ get used in [this change](https://github.com/Adjective-Object/good-fences/commit/722050c9b7483fc7854e6af946ba8f37ca859c2f) which depends on #97, and I left it exported here as an oversight. I'll just include it in the change that uses it.